.github/workflows/build.yml

```diff
@@ -142,6 +142,7 @@ jobs:
         run: |
           make yarn-install
           make only-install-server
+          make install-examples
```

Comment:
in order to match our normal build order I would put this in the Setup JupyterLab section below